tests/_support/Traits/With_Uopz.php

```diff
@@ -104,4 +104,22 @@ private function set_class_fn_return( $class, $method, $value, $execute = false
 		uopz_set_return( $class, $method, $value, $execute );
 		$this->uopz_set_returns[] = [ $class, $method ];
 	}
+
+	private function set_class_property( $object, $field, $value ) {
+		if ( ! function_exists( 'uopz_set_property' ) ) {
+			$this->markTestSkipped( 'uopz extension is not installed' );
+		}
+		uopz_set_property( $object, $field, $value );
+	}
+
+	private function add_class_fn( $class, $function, $handler ) {
+		if ( ! function_exists( 'uopz_add_function' ) ) {
+			$this->markTestSkipped( 'uopz extension is not installed' );
+		}
+		uopz_add_function(
```

Comment:
As the comment above, the trait should define an `@after` method to unset at method set for the purpose of testing.

Reply:
Added some unset/cleanup. Not all API's have a uopz unset, so I got creative.